documentation/configuration.markdown

```diff
@@ -5,558 +5,42 @@ canonical: "/puppetserver/latest/configuration.html"
-For more detailed information on the format of this file, see
-[Configuring the Authorization Service](https://github.com/puppetlabs/trapperkeeper-authorization/blob/master/doc/authorization-config.md).
-
-#### Creating custom rules
-
-If you need to customize the authorization rules for Puppet Server, it is
-recommended that you create new rules rather than customizing the default
-"puppetlabs" rules which appear in this file.  In order for your rules to
-"override" any corresponding "puppetlabs" rules, you should use a
-`sort-order` for those rules which is in the range of 1 to 399 (inclusive).
-Note that default rules from Puppet occupy the range from 400 to 600 (inclusive).
-
-For example, if you wanted to customize the behavior of the default "catalog"
-rule from above to not only allow nodes to retrieve their own catalog but also
-allow an "administrative" node to retrieve any node's catalog, you could add
-a rule like this:
-
-~~~
-authorization: {
-    version: 1
-    # allow-header-cert-info: false
-    rules: [
-        {
-            # Allow nodes to retrieve their own catalog
-            # and admin nodes to retrieve any catalogs
-            match-request: {
-                path: "^/puppet/v3/catalog/([^/]+)$"
-                type: regex
-                method: [get, post]
-            }
-            allow: ["$1", "myadmin.host.com"]
-            sort-order: 200
-            name: "my catalog"
-        },
-        {
-            # Allow nodes to retrieve their own catalog
-            match-request: {
-                path: "^/puppet/v3/catalog/([^/]+)$"
-                type: regex
-                method: [get, post]
-            }
-            allow: "$1"
-            sort-order: 500
-            name: "puppetlabs catalog"
-        },
-...
-~~~
-
-If you want to add a rule but let the default rules from Puppet take
-precedence over your new rule, you should use a `sort-order` for the rule
-which is in the range from 601 to 998 (inclusive).
-
-#### Legacy Puppet auth.conf compatibility considerations
-
-In general, the functionality provided for `trapperkeeper-authorization`
-"auth.conf"  rules is comparable to rule definitions with the core
-[Puppet auth.conf][auth.conf] file.  This section details some differences
-around rule definitions to keep in mind when migrating custom rules to the
-`trapperkeeper-authorization` "auth.conf" format.
-
-##### `path`
-
-For the `path` directive, core Puppet's "auth.conf" would use the presence of
-a tilde (`~`) character to distinguish a path representing a regex vs. a literal
-string which the endpoint must start with in order to be a match.
-
-For example:
-
-~~~~
-#regex
-path ~ ^/puppet/v3/report/([^/]+)$
-
-#literal string
-path /puppet/v3/report/
-~~~~
-
-For a `trapperkeeper-authorization` "auth.conf" rule, the distinction between
-a regex vs. a literal "starts-with" string is done via the use of an explicit
-`type` attribute in the rule.
-
-For example:
+#### Example (Legacy)
```

Comment:
Would it be better for this example to move over to the new `config_file_ca.html` file?